vm/systemSmartContracts/delegation.go

```diff
@@ -2432,6 +2439,21 @@ func (d *delegation) getMinDeposit() (*big.Int, error) {
 	return managementData.MinDeposit, nil
 }
 
+func (d *delegation) getMinDelegationAmount() (*big.Int, error) {
+	marshaledData := d.eei.GetStorageFromAddress(d.delegationMgrSCAddress, []byte(delegationManagementKey))
```

Comment:
duplicated code with getMinDeposit. make a separate function for get delegationManagement config.

Reply:
removed duplicated code, added unit tests.